README.md

```diff
@@ -35,6 +35,8 @@ The bootstrap script can also rebuild your Docker images and spin them up althou
     2. `docker-compose up --force-recreate --remove-orphans` will re-create all containers known to docker-compose and delete those volumes that no longer are associated with running containers
     3. `docker system prune` for cleaning out old containers and images
 
+If using with logstash, use `-f docker-compose.yml -f docker-compose.logstash.yml` flags after each `docker-compose` command to merge docker-compose files
```

Comment:
docker-compose.logstash.yml could probably just be docker-compose.developer.yml or docker-compose.dev.yml and basically grow to encompass stuff that developers may want to run a self-contained dockstore install but would normally be running on a different computer in production/staging